scripts/update_genesis_file.sh

```diff
@@ -15,70 +15,16 @@ jq '.consensus_params["block"]["max_bytes"] = "3145728"' "$GENESIS_FILE" >"$tmp"
-          "account_number": "0",
-          "address": $address,
-          "pub_key": null,
-          "sequence": "0"
-      },
-      "name": "hubgenesis",
-      "permissions": []
-  }]')
-
-jq --argjson module_account "$module_account" '.app_state.auth.accounts += $module_account' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
-
-module_account_balance=$(
-  jq -n \
-    --arg address "$module_account_address" \
-    --arg denom "$BASE_DENOM" \
-    '[{
-    "address": $address,
-    "coins": [
-      {
-        "denom": $denom,
-        "amount": "60000000000000000000000"
-      }
-    ]
-  }]'
-)
-
-hubgenesis_tokens=$(
-jq -n \
-  --arg denom "$BASE_DENOM" \
-  --arg amount "60000000000000000000000" \
-  '[{
-    "denom": $denom,
-    "amount": $amount
-  }]'
-)
-
-jq --argjson hubgenesis_tokens "$hubgenesis_tokens" '.app_state.hubgenesis.state.genesis_tokens = $hubgenesis_tokens' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
-
 jq '.app_state.bank.balances[0].coins[0].amount = "2000000000000000000000000000"' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
-jq --argjson module_account_balance "$module_account_balance" '.app_state.bank.balances += $module_account_balance' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
-
-jq '.app_state.bank.supply[0].amount = "2000060000000000000000000000"' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
+jq '.app_state.bank.supply[0].amount = "2000000000000000000000000000"' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
 
-# ---------------------------- add elevated account ---------------------------- #
-elevated_address=$("$EXECUTABLE" keys show "$KEY_NAME_ROLLAPP" --keyring-backend test --output json | jq -r .address)
-elevated_address_json=$(jq -n \
-  --arg address "$elevated_address" \
-  '[{
-        "address": $address
-    }]')
-jq --argjson elevated_address_json "$elevated_address_json" '.app_state.hubgenesis.params.genesis_triggerer_allowlist += $elevated_address_json' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
+# ---------------------------- add genesis accounts for the hub ---------------------------- #
+genesis_accounts=$(cat "$ROLLAPP_SETTLEMENT_INIT_DIR_PATH"/genesis_accounts.json)
+jq --argjson genesis_accounts "$genesis_accounts" '.app_state.hubgenesis.state.genesis_accounts = $genesis_accounts' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
 
 # ---------------------------- add denom metadata ---------------------------- #
 denom_metadata=$(cat "$ROLLAPP_SETTLEMENT_INIT_DIR_PATH"/denommetadata.json)
+elevated_address=$("$EXECUTABLE" keys show "$KEY_NAME_ROLLAPP" --keyring-backend test --output json | jq -r .address)
 jq --argjson denom_metadata "$denom_metadata" '.app_state.bank.denom_metadata = $denom_metadata' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
 jq --arg elevated_address "$elevated_address" '.app_state.denommetadata.params.allowed_addresses += [$elevated_address]' "$GENESIS_FILE" >"$tmp" && mv "$tmp" "$GENESIS_FILE"
```

Comment:
I don't think we have `denom_metadata.params...` anymore

Reply:
do we or don't we?

Reply:
I'll leave it to you to finish this please @zale144 
As it corresponds to your denom meta data work